### PR TITLE
[NUI][AT-SPI] Set focus in OnAccessibilityActivated()

### PR DESCRIPTION
**Pull request by aswigon**
### Description of Change ###

This aligns the behaviour of `Components` (derived from `CustomView`), and other controls, with that of `BaseComponents` (and other controls with `ViewAccessibilityMode.Default`, backed by the C++ accessibility implementation), which set the keyboard focus on activation. Please note that in the latter case, the implementation of `View.OnAccessibilityActivated` is unused, and the unrelated method (albeit bearing the same name) `Control::OnAccessibilityActivated` is called:

```cpp
bool Control::OnAccessibilityActivated()
{
  if(Toolkit::KeyboardFocusManager::Get().SetCurrentFocusActor(Self()))
  {
    return OnKeyboardEnter();
  }
  return false;
}
```

So, in other words, this patch makes C# controls behave in a similar manner, i.e. set the keyboard focus on accessibility activation.

### API Changes ###

 - ACR: None

### CODE REVIEW

**Review by loleksak — APPROVED** [review]
LGTM, now its consistent with other toolkits

**Review by Seoyeon2Kim — APPROVED** [review]
I think it's Okay because the similar implementation is already applied to dali-toolkit one. :)

**Review by JoogabYun — APPROVED** [review]
